src/App.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
```

Comment:
You don't need to use app.jsx for testing your redux you can create another components for testing purpose because this components is going to work with our react router setting

Reply:
Thanks. Fixed